src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    static final int NAME_INDEX = 0;
+    static final int PARAM_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeWtoFile(createReport(readFromFile(fromFileName)), toFileName);
+    }
+
+    public List<String> readFromFile(String fromFileName) {
+        List<String> fileLines;
+        ArrayList<String> columnList;
+        try {
+            fileLines = Files.readAllLines(Paths.get(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file: " + fromFileName, e);
+        }
+        return fileLines;
+    }
+
+    public Integer[] createReport(List<String> fileLines) {
+        int supplySum = 0;
+        int buySum = 0;
+        for (String fileLine : fileLines) {
+            String[] splitedText = fileLine.split(",");
+            ArrayList<String> columnList = new ArrayList<>(Arrays.asList(splitedText));
+            if (columnList.get(NAME_INDEX).equals("supply")) {
+                supplySum += Integer.parseInt(columnList.get(PARAM_INDEX));
+            } else {
+                buySum += Integer.parseInt(columnList.get(PARAM_INDEX));
+            }
+        }
+        return new Integer[]{supplySum, buySum};
```

Comment:
make a report and return it from this method. Your method is called createReport - so it should return a ready report

Reply:
https://mate-parttime.slack.com/archives/C027GNX3DFW/p1687456179766069